analyzers/rspec/cs/S6422_c#.html

```diff
@@ -0,0 +1,73 @@
+<p>Making blocking calls to <code>async</code> methods transforms something that was intended to be asynchronous into a synchronous block. Doing so
```

Comment:
"something" and "synchronous block" sound weird. Maybe like so?
```suggestion
<p>Making blocking calls to <code>async</code> methods transforms code that was intended to be asynchronous into a blocking operation. Doing so
```